test/browser/weighted.test.jsx

```diff
@@ -55,7 +54,8 @@ describe('Weighted Experiment', () => {
     });
 
     for (let i = 0; i < 1000; i++) {
-      mount(<App />);
+      const container = mount(<App />);
+      container.unmount();
```

Comment:
looks like memory pressure was what was making this test take a long time to complete